src/api/web/index.js

```diff
@@ -43,6 +43,25 @@ module.exports = function(config, auth, storage) {
   const indexTemplate = path.join(themePath, 'index.html');
   const template = fs.readFileSync(indexTemplate).toString();
 
+  // Logo
+  let logoURI = _.get(config, 'web.logo') ? config.web.logo : '';
+  if (logoURI && !/^(https?:)?\/\//.test(logoURI)) {
+    // URI that not starts with "http://", "https://" or "//"
+    logoURI = path.join('/-/static/', path.basename(logoURI));
+    router.get(logoURI, function(req, res, next) {
+      res.sendFile(path.resolve(config.web.logo), function(err) {
```

Comment:
I like your solution, but we need to clean up this a bit. The following callback is duplicated in the same file. 
```
function(err) {
        if (!err) {
          return;
        }
        if (err.status === HTTP_STATUS.NOT_FOUND) {
          next();
        } else {
          next(err);
        }
      }
```

Reply:
Done.